docs/how-to/distributed-training.txt

```diff
@@ -142,6 +142,30 @@ distributed training:
    can also be used to check if any other tasks are using GPUs and
    preventing your experiment from using all the GPUs on a machine.
 
+***********************
+ Distributed Inference
+***********************
+
+PyTorch users can also use the existing distributed training workflow
+with PytorchTrial to accelerate their inference workloads. This workflow
```

Comment:
let's point to the inference example in the doc here.

Reply:
Good catch, added